src/python/library/tritonclient/utils/cuda_shared_memory/__init__.py

```diff
@@ -173,20 +181,27 @@ def set_shared_memory_region(cuda_shm_handle, input_values):
                 "input_values must be specified as a list/tuple of numpy arrays"
```

Comment:
Need to expand checks here to allow for a single cuda device ptr instead  of a list or tuple of numpy arrays